Packs/ServiceNowv2/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -0,0 +1,1542 @@
+    if 'opened_by' in data:
+        context['OpenedBy'] = data['opened_by']['value'] if 'value' in data['opened_by'] else ''
+        context['Creator'] = data['opened_by']['value'] if 'value' in data['opened_by'] else ''
+    if 'assigned_to' in data:
+        context['Assignee'] = data['assigned_to']['value'] if 'value' in data['assigned_to'] else ''
+
+    # Try to map fields
+    if 'priority' in data:
+        context['Priority'] = TICKET_PRIORITY.get(data['priority'], data['priority'])
+    if 'state' in data:
+        context['State'] = data['state']
+
+    return createContext(context, removeNull=True)
+
+
+def get_ticket_context(data: Any) -> Any:
+    """Manager of ticket context creation.
+
+    Args:
+        data: ticket data. in the form of a dict or a list of dict.
+
+    Returns:
+        ticket context. in the form of a dict or a list of dict.
+    """
+    if not isinstance(data, list):
+        return create_ticket_context(data)
+
+    tickets = []
+    for d in data:
+        tickets.append(create_ticket_context(d))
+    return tickets
+
+
+def get_ticket_human_readable(tickets, ticket_type: str) -> list:
+    """Get ticket human readable.
+
+    Args:
+        tickets: tickets data. in the form of a dict or a list of dict.
+        ticket_type: ticket type.
+
+    Returns:
+        ticket human readable.
+    """
+    if not isinstance(tickets, list):
+        tickets = [tickets]
+
+    ticket_severity = {
+        '1': '1 - High',
+        '2': '2 - Medium',
+        '3': '3 - Low'
+    }
+
+    result = []
+    for ticket in tickets:
+
+        hr = {
+            'Number': ticket.get('number'),
+            'System ID': ticket['sys_id'],
+            'Created On': ticket.get('sys_created_on'),
+            'Created By': ticket.get('sys_created_by'),
+            'Active': ticket.get('active'),
+            'Close Notes': ticket.get('close_notes'),
+            'Close Code': ticket.get('close_code'),
+            'Description': ticket.get('description'),
+            'Opened At': ticket.get('opened_at'),
+            'Due Date': ticket.get('due_date'),
+            # This field refers to a record in the database, the value is its system ID.
+            'Resolved By': ticket.get('closed_by', {}).get('value') if isinstance(ticket.get('closed_by'), dict)
+            else ticket.get('closed_by'),
+            'Resolved At': ticket.get('resolved_at'),
+            'SLA Due': ticket.get('sla_due'),
+            'Short Description': ticket.get('short_description'),
+            'Additional Comments': ticket.get('comments')
+        }
+
+        # Try to map the fields
+        if 'impact' in ticket:
+            hr['Impact'] = ticket_severity.get(ticket['impact'], ticket['impact'])
+        if 'urgency' in ticket:
+            hr['Urgency'] = ticket_severity.get(ticket['urgency'], ticket['urgency'])
+        if 'severity' in ticket:
+            hr['Severity'] = ticket_severity.get(ticket['severity'], ticket['severity'])
```

Comment:
is ticket severity is the same 3 values for impact and urgency by default?
and also, it is probably safer to use get one time for every value and use it.
like that:
```
impact = ticket.get('impact', '')
hr['Impact'] = ticket_severity.get(impact, impact)
```

Reply:
Yes to the first question.